src/Public/OpenApi.ps1

```diff
@@ -2695,6 +2713,8 @@ function New-PodeOASchemaProperty {
             if ($Xml) {
                 $param.xml = $Xml
             }
+        } elseif ($Description) {
+            Write-PodeHost "New-PodeOASchemaProperty $ComponentSchema - Description can only be applied to an array"
```

Comment:
Would this be better to throw as an error instead?

Reply:
yes but I break the compatibility with previous version 
I'll love to log it but I was not able to find how to achieve it

Reply:
Aah, good point. In which case the following should log it as a warning:
```powershell
$ex = [System.Exception]::new("New-PodeOASchemaProperty $ComponentSchema - Description can only be applied to an array")
$ex | Write-PodeErrorLog -Level Warning
```